src/components/Container/Container.less

```diff
@@ -0,0 +1,27 @@
+@horizontal-lines: 1px solid @color-gray;
```

Comment:
These variable names aren't quite right. See the [contributing doc styling section](https://github.com/appnexus/lucid/blob/master/CONTRIBUTING.md#styling) for more examples of less variable names.